### ISSUE TITLE
⌨ Descriptive label for functional communities missing

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13342]
### 🛡️Fable tester: Michael S. 
The button to view and edit the financial management community settings does not have a descriptive label. The button is labeled with the name of the community but it does not say what will happen When I click the button.

### Expected behaviour: 
The button should tell you that you are going to view the details

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

VoiceOver

### 📸 Screenshot

<img width="1441" alt="Image" src="https://github.com/user-attachments/assets/133a25ee-0eec-4f37-940a-ef14b9bc6e82" />

### ✅ Acceptance Criteria

- [ ] accessible labels added for community interest dialog triggers
  - [ ] View your {communityName} interests
  - [ ] Voir vos intérêts pour la {communityName}

**Comment by petertgiles** [comment]
@BrookeCameron what should the descriptive label be?  Remember that it will have to be unique in a list, as well.

**Comment by BrookeCameron** [comment]
I think "View "community name here" details" ... what do you think @NienkeBr ?

**Comment by NienkeBr** [comment]
We could stay close to the subtitle of the dialogue and say:
- View your {communityName} settings
- Voir vos paramètres pour la {communityName}

**Comment by petertgiles** [comment]
Discussion in standup: Nienke will update the label from "settings" to "interest"
